.pipelines/.vsts-vhd-builder-release-windows.yaml

```diff
@@ -56,6 +56,10 @@ parameters:
     displayName: Build 2025 Gen 2
     type: boolean
     default: True
+  - name: windowsContainerImageJsonUrl
+    displayName: Windows Container Image Json URL
+    type: string
+    default: https://wcctagentbakerstorage.blob.core.windows.net/simship/2025/06B/payload.json
```

Comment:
This default should be an empty string or else all runs of this pipeline will use the hard coded value.